manufacturing-ai-suite/wind-turbine-anomaly-detection/deployment-package/timeseries-wind-turbine-registry.yaml

```diff
@@ -0,0 +1,121 @@
+#
+# Apache v2 license
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+specSchema: "Registry"
+schemaVersion: "0.1"
+$schema: "https://schema.intel.com/catalog.orchestrator/0.1/schema"
+
+name: ts-wind-turbine-anomaly-registry
+description: OCI registry for Wind Turbine Anomaly Detection Sample App
+type: HELM
+rootUrl: oci://amr-registry.caas.intel.com/edge-insights/timeseries/
```

Comment:
Do not use Intel internal links.

Reply:
Thanks for catching, we had done this to simplify the app onboarding on EMF orch but now that we are putting in open, we will have this put in the documentation.